compiler/optimizer/OMRSimplifierHandlers.hpp

```diff
@@ -267,5 +267,5 @@ TR::Node * NewSimplifier(TR::Node * node, TR::Block * block, TR::Simplifier * s)
 TR::Node * lowerTreeSimplifier(TR::Node * node, TR::Block * block, TR::Simplifier * s);
 TR::Node * arrayLengthSimplifier(TR::Node * node, TR::Block * block, TR::Simplifier * s);
 
-
+TR::Node * reduceIntegerCompareArithmetics(TR::Node* node,TR::Simplifier * s);
```

Comment:
Not a big fan of the name - this almost seems like a form of comparison strength reduction (see https://en.wikipedia.org/wiki/Strength_reduction), but I'm not sure it is truly strength reduction. A better, more descriptive name is certainly warranted.